.github/workflows/ipfs-deploy.yml

```diff
@@ -65,3 +65,71 @@ jobs:
       - name: Publish to nft.storage
         run: |
           docker run -e NFTSTORAGE_API_KEY=${{ secrets.NFTSTORAGE_API_KEY }} $IMAGE_RELEASE_ID nft.storage
+
+      - name: Create a release
+        if: startsWith(github.ref, 'refs/tags/')
+        env:
+          REF_NAME: ${{ github.ref_name }}
+          GITHUB_REPOSITORY: ${{ github.repository }}
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          # Markdown template for the release notes
+          RELEASE_NOTE_TEMPLATE=$(cat << EOF
+          #### IPFS Hash
+
+          \`\`\`
+          $IPFS_HASH
+          \`\`\`
+
+          You can view published versions of Lunaria through any IPFS Gateway
+
+          [ipfs://$IPFS_HASH](ipfs://$IPFS_HASH) __(Recommended)__
+          _requires Brave Browser or IPFS Desktop_
+          [https://$IPFS_HASH.ipfs.nftstorage.link](https://$IPFS_HASH.ipfs.nftstorage.link)
+          [https://$IPFS_HASH.ipfs.zoltu.io](https://$IPFS_HASH.ipfs.zoltu.io)
+          [https://$IPFS_HASH.ipfs.keydonix.com](https://$IPFS_HASH.ipfs.keydonix.com)
+          [https://$IPFS_HASH.ipfs.cf-ipfs.com](https://$IPFS_HASH.ipfs.cf-ipfs.com)
+          [https://$IPFS_HASH.ipfs.w3s.link](https://$IPFS_HASH.ipfs.w3s.link)
+          EOF
+          )
+
+          # Generate payload for creating a new release
+          PAYLOAD_TEMPLATE=$(cat <<EOF
+          {
+            "name": "$REF_NAME",
+            "tag_name": "$REF_NAME",
+            "target_commitish": "main",
```

Comment:
I don't think this is correct?  We are releasing from tags, not from `main`.  I'm not sure what should be here, but it feels like `main` is wrong.

Reply:
Removed the payload value as it's not needed as the tag is already created.

<img width="579" alt="Screenshot 2024-02-27 at 2 58 03 PM" src="https://github.com/DarkFlorist/lunaria/assets/1169838/0b909e7f-d00f-4ed5-96a3-113515f0e3b6">

https://docs.github.com/en/rest/releases/releases?apiVersion=2022-11-28